backend/graphql/resolvers/authResolvers.ts

```diff
@@ -70,6 +74,23 @@ const authResolvers = {
       await authService.resetPassword(email);
       return true;
     },
+    sendEmailVerificationLink: async (
+      _parent: undefined,
+      { email }: { email: string },
+    ): Promise<boolean> => {
+      try {
+        await authService.sendEmailVerificationLink(email);
+        return true;
+      } catch (error: unknown) {
+        const errorMessage = [
+          `Failed to resolve email verification link mutation for user with email ${email}`,
```

Comment:
If these strings are being joined with a " ", it might make sense to add a trailing comma to this line